### ISSUE TITLE
Active ToR Link Down Test Cases

### ISSUE THREAD

**Issue by theasianpianist** [Azure/sonic-mgmt#2763]
Test all packet flow scenarios after link between the active ToR and fanout goes down.

Depends on #2758, #2759, #2756

**Comment by theasianpianist** [comment]
Putting on hold for now, since there is not a good way to blackhole heartbeat traffic
